mtwitter.js

```diff
@@ -0,0 +1,291 @@
+var Twitter = require('twitter');
+var Request = require('request');
+var fs = require('fs');
+var Buffer = require('buffer').Buffer;
+
+const TWITTER_PROFILE_INTERVAL = 60000;//ms;
+
+var MTwitter = function (bridge, config) {
+  this.app_auth = config.app_auth;
+  this.app_twitter = null;
+  this.tuser_cache = {};
+  this.timeline_list = [];
+  this.timeline_queue = [];
+  this.timeline_period = 0;
+  this.timeline_intervalID = null;
+  this.msg_queue = [];
+  this._bridge = bridge;
+
+
+  this.get_bearer_token().then((token) => {
```

Comment:
As I said before, it is slightly surprising when constructing a new object results in big side effects like making HTTP hits, writing to disk and starting a long-running poll. You should probably have a `start()` method for this.